afs/media/js/views/components/reports/textual-work.js

```diff
@@ -37,38 +41,160 @@ define([
             self.documentationCards = {};
             self.existenceEvents = ['creation'];
             self.existenceDataConfig = {
-                'creation': 'Creation (partitioned)',
+                creation: {
+                    graph: 'creation (partitioned)',
+                    metadata: [{
+                        key: 'Creation Event Type',
+                        path: 'creation (partitioned)_type',
+                        type: 'kv'
+                    },{
+                        key: 'Creator of Textual Work',
+                        path: 'creation (partitioned)_carried out by',
+                        type: 'resource'
+                    },{
+                        key: 'Creation Event Location',
+                        path: 'creation (partitioned)_location',
+                        type: 'resource'
+                    }],
+                    parts: {
+                        graph: 'creation (partitioned)_part',
+                        metadata:[{
+                            key: 'Creation Event Part Type',
+                            path: 'creation (partitioned)_part_type',
+                            type: 'kv'
+                        },{
+                            key: 'Creator in Creation Event Part',
+                            path: 'creation (partitioned)_part_carried out by',
+                            type: 'resource'
+                        },{
+                            key: 'Creation Event Part Location',
+                            path: 'creation (partitioned)_part_location',
+                            type: 'resource'
+                        }]
+                    }
+                }
             };
             self.existenceCards = {};
+            self.eventEvents = ['publication'];
+            self.eventDataConfig = {
+                publication: {
+                    graph: 'publication',
+                    metadata: [{
+                        key: 'Publication Event Type',
+                        path: 'Publication_type',
+                        type: 'kv'
+                    },{
+                        key: 'Particular Technique',
+                        path: 'Publication_technique',
+                        type: 'kv'
+                    },{
+                        key: 'Publication Event Location',
+                        path: 'Publication_location',
+                        type: 'resource'
+                    }]
+                }
+            };
+            self.eventCards = {};
+            self.substanceCards = {};
+
             self.summary = params.summary;
 
             if(params.report.cards){
                 const cards = params.report.cards;
                 
-                self.cards = self.createCardDictionary(cards)
+                self.cards = self.createCardDictionary(cards);
                 self.nameCards = {
                     name: self.cards?.['Textual Work Name'],
```

Comment:
noticed that the name cards are incorrect.

Reply:
thanks. All the cards and nodes name are updated